src/blueprint/common/evmContractReader.ts

```diff
@@ -1,11 +1,15 @@
+import { executeCallOrSend } from '../..//web3-wrapper';
+import { RpcInfo } from '../../web3-wrapper/rpc/rpcInfo';
 import { BlueprintContext } from '../models';
 import { formatAsDecimalAwareString } from './utils';
 import BigNumber from 'bignumber.js';
 import { ethers } from 'ethers';
-import { executeCallOrSend } from '../..//web3-wrapper';
 
 // Please set your favorable and reliable RPC URLs
-const rpcUrls = ['https://polygon-rpc.com', 'https://rpc.ankr.com/polygon'];
+const rpcs: RpcInfo[] = [
+  { url: 'https://polygon-rpc.com', weight: 1 },
+  { url: 'https://rpc.ankr.com/polygon', weight: 1 },
```

Comment:
we don't need this since this should come dynamically

Reply:
I don't find the usage of these class anywhere for now, but since it is the legacy code, I think we should not delete it, what's your optinion?

Reply:
can we delete this in this MR? It's confusing for someone reading this code to understand why this is in here